2022-05-26 11:13:17.524  INFO 7296 --- [restartedMain] c.e.addressbook.AddressBookApplication   : Starting AddressBookApplication using Java 18.0.1.1 on Vishaka with PID 7296 (D:\SpringBoot\AddressBook\target\classes started by ACER in D:\SpringBoot\AddressBook)
2022-05-26 11:13:17.526 DEBUG 7296 --- [restartedMain] c.e.addressbook.AddressBookApplication   : Running with Spring Boot v2.7.0, Spring v5.3.20
2022-05-26 11:13:17.528  INFO 7296 --- [restartedMain] c.e.addressbook.AddressBookApplication   : The following 1 profile is active: "dev"
2022-05-26 11:13:18.349  INFO 7296 --- [restartedMain] c.e.addressbook.AddressBookApplication   : Started AddressBookApplication in 1.045 seconds (JVM running for 1.421)
2022-05-26 11:13:18.352  INFO 7296 --- [restartedMain] c.e.addressbook.AddressBookApplication   : AddressBook App Started in DEV Environment
2022-05-26 11:13:18.352  INFO 7296 --- [restartedMain] c.e.addressbook.AddressBookApplication   : AddressBook App DB User is root
2022-05-26 11:13:59.497  INFO 12628 --- [restartedMain] c.e.addressbook.AddressBookApplication   : Starting AddressBookApplication using Java 18.0.1.1 on Vishaka with PID 12628 (D:\SpringBoot\AddressBook\target\classes started by ACER in D:\SpringBoot\AddressBook)
2022-05-26 11:13:59.499 DEBUG 12628 --- [restartedMain] c.e.addressbook.AddressBookApplication   : Running with Spring Boot v2.7.0, Spring v5.3.20
2022-05-26 11:13:59.499  INFO 12628 --- [restartedMain] c.e.addressbook.AddressBookApplication   : The following 1 profile is active: "dev"
2022-05-26 11:14:00.343  INFO 12628 --- [restartedMain] c.e.addressbook.AddressBookApplication   : Started AddressBookApplication in 1.224 seconds (JVM running for 1.771)
2022-05-26 11:14:00.346  INFO 12628 --- [restartedMain] c.e.addressbook.AddressBookApplication   : AddressBook App Started in DEV Environment
2022-05-26 11:14:00.346  INFO 12628 --- [restartedMain] c.e.addressbook.AddressBookApplication   : AddressBook App DB User is root
2022-05-26 11:18:03.934  INFO 11232 --- [restartedMain] c.e.addressbook.AddressBookApplication   : Starting AddressBookApplication using Java 18.0.1.1 on Vishaka with PID 11232 (D:\SpringBoot\AddressBook\target\classes started by root in D:\SpringBoot\AddressBook)
2022-05-26 11:18:03.937 DEBUG 11232 --- [restartedMain] c.e.addressbook.AddressBookApplication   : Running with Spring Boot v2.7.0, Spring v5.3.20
2022-05-26 11:18:03.937  INFO 11232 --- [restartedMain] c.e.addressbook.AddressBookApplication   : The following 1 profile is active: "dev"
2022-05-26 11:18:04.737  INFO 11232 --- [restartedMain] c.e.addressbook.AddressBookApplication   : Started AddressBookApplication in 1.012 seconds (JVM running for 1.377)
2022-05-26 11:18:04.737  INFO 11232 --- [restartedMain] c.e.addressbook.AddressBookApplication   : AddressBook App Started in DEV Environment
2022-05-26 11:18:04.737  INFO 11232 --- [restartedMain] c.e.addressbook.AddressBookApplication   : AddressBook App DB User is root
2022-05-26 11:19:00.146  INFO 3492 --- [restartedMain] c.e.addressbook.AddressBookApplication   : Starting AddressBookApplication using Java 18.0.1.1 on Vishaka with PID 3492 (D:\SpringBoot\AddressBook\target\classes started by root in D:\SpringBoot\AddressBook)
2022-05-26 11:19:00.147 DEBUG 3492 --- [restartedMain] c.e.addressbook.AddressBookApplication   : Running with Spring Boot v2.7.0, Spring v5.3.20
2022-05-26 11:19:00.147  INFO 3492 --- [restartedMain] c.e.addressbook.AddressBookApplication   : The following 1 profile is active: "dev"
2022-05-26 11:19:00.933  INFO 3492 --- [restartedMain] c.e.addressbook.AddressBookApplication   : Started AddressBookApplication in 1.09 seconds (JVM running for 1.676)
2022-05-26 11:19:00.948  INFO 3492 --- [restartedMain] c.e.addressbook.AddressBookApplication   : AddressBook App Started in DEV Environment
2022-05-26 11:19:00.948  INFO 3492 --- [restartedMain] c.e.addressbook.AddressBookApplication   : AddressBook App DB User is root
2022-05-26 11:31:35.088  INFO 7228 --- [restartedMain] c.e.addressbook.AddressBookApplication   : Starting AddressBookApplication using Java 18.0.1.1 on Vishaka with PID 7228 (D:\SpringBoot\AddressBook\target\classes started by root in D:\SpringBoot\AddressBook)
2022-05-26 11:31:35.090 DEBUG 7228 --- [restartedMain] c.e.addressbook.AddressBookApplication   : Running with Spring Boot v2.7.0, Spring v5.3.20
2022-05-26 11:31:35.090  INFO 7228 --- [restartedMain] c.e.addressbook.AddressBookApplication   : The following 1 profile is active: "dev"
2022-05-26 11:31:35.882  INFO 7228 --- [restartedMain] c.e.addressbook.AddressBookApplication   : Started AddressBookApplication in 1.017 seconds (JVM running for 1.415)
2022-05-26 11:31:35.882  INFO 7228 --- [restartedMain] c.e.addressbook.AddressBookApplication   : AddressBook App Started in DEV Environment
2022-05-26 11:31:35.882  INFO 7228 --- [restartedMain] c.e.addressbook.AddressBookApplication   : AddressBook App DB User is root
2022-05-26 11:32:25.117  WARN 7228 --- [http-nio-8088-exec-1] .w.s.m.s.DefaultHandlerExceptionResolver : Resolved [org.springframework.web.bind.MethodArgumentNotValidException: Validation failed for argument [0] in public org.springframework.http.ResponseEntity<com.example.addressbook.dto.ResponseDTO> com.example.addressbook.controller.AddressBookController.createContactData(com.example.addressbook.dto.ContactDTO) with 5 errors: [Field error in object 'contactDTO' on field 'email': rejected value [pihu123.gmail.com]; codes [Pattern.contactDTO.email,Pattern.email,Pattern.java.lang.String,Pattern]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [contactDTO.email,email]; arguments []; default message [email],[Ljavax.validation.constraints.Pattern$Flag;@6ab6010c,^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9]+(?:\.[a-zA-Z0-9]+)*$]; default message [must match "^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9]+(?:\.[a-zA-Z0-9]+)*$"]] [Field error in object 'contactDTO' on field 'phone': rejected value [7593450]; codes [Pattern.contactDTO.phone,Pattern.phone,Pattern.java.lang.String,Pattern]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [contactDTO.phone,phone]; arguments []; default message [phone],[Ljavax.validation.constraints.Pattern$Flag;@6ab6010c,^[0-9]{2}?[\s,-]{0,1}[7-9]{1}[0-9]{9}$]; default message [must match "^[0-9]{2}?[\s,-]{0,1}[7-9]{1}[0-9]{9}$"]] [Field error in object 'contactDTO' on field 'zip': rejected value [5000]; codes [Pattern.contactDTO.zip,Pattern.zip,Pattern.java.lang.String,Pattern]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [contactDTO.zip,zip]; arguments []; default message [zip],[Ljavax.validation.constraints.Pattern$Flag;@6ab6010c,^[0-9]{3}\s{0,1}[0-9]{3}$]; default message [must match "^[0-9]{3}\s{0,1}[0-9]{3}$"]] [Field error in object 'contactDTO' on field 'firstName': rejected value [shanvika]; codes [Pattern.contactDTO.firstName,Pattern.firstName,Pattern.java.lang.String,Pattern]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [contactDTO.firstName,firstName]; arguments []; default message [firstName],[Ljavax.validation.constraints.Pattern$Flag;@6ab6010c,^[A-Z]{1}[a-zA-Z\s]{2,}$]; default message [Address book is FirstNameInvalid ]] [Field error in object 'contactDTO' on field 'lastName': rejected value [jk]; codes [Pattern.contactDTO.lastName,Pattern.lastName,Pattern.java.lang.String,Pattern]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [contactDTO.lastName,lastName]; arguments []; default message [lastName],[Ljavax.validation.constraints.Pattern$Flag;@6ab6010c,^[A-Z]{1}[a-zA-Z\s]{2,}$]; default message [Address book is LastNameInvalid]] ]
2022-05-26 11:40:50.590  INFO 9360 --- [restartedMain] c.e.addressbook.AddressBookApplication   : Starting AddressBookApplication using Java 18.0.1.1 on Vishaka with PID 9360 (D:\SpringBoot\AddressBook\target\classes started by root in D:\SpringBoot\AddressBook)
2022-05-26 11:40:50.592 DEBUG 9360 --- [restartedMain] c.e.addressbook.AddressBookApplication   : Running with Spring Boot v2.7.0, Spring v5.3.20
2022-05-26 11:40:50.593  INFO 9360 --- [restartedMain] c.e.addressbook.AddressBookApplication   : The following 1 profile is active: "dev"
2022-05-26 11:40:51.387  INFO 9360 --- [restartedMain] c.e.addressbook.AddressBookApplication   : Started AddressBookApplication in 1.009 seconds (JVM running for 1.403)
2022-05-26 11:40:51.387  INFO 9360 --- [restartedMain] c.e.addressbook.AddressBookApplication   : AddressBook App Started in DEV Environment
2022-05-26 11:40:51.387  INFO 9360 --- [restartedMain] c.e.addressbook.AddressBookApplication   : AddressBook App DB User is root
2022-05-26 11:41:01.407  WARN 9360 --- [http-nio-8088-exec-6] .m.m.a.ExceptionHandlerExceptionResolver : Resolved [org.springframework.web.bind.MethodArgumentNotValidException: Validation failed for argument [0] in public org.springframework.http.ResponseEntity<com.example.addressbook.dto.ResponseDTO> com.example.addressbook.controller.AddressBookController.createContactData(com.example.addressbook.dto.ContactDTO) with 5 errors: [Field error in object 'contactDTO' on field 'zip': rejected value [5000]; codes [Pattern.contactDTO.zip,Pattern.zip,Pattern.java.lang.String,Pattern]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [contactDTO.zip,zip]; arguments []; default message [zip],[Ljavax.validation.constraints.Pattern$Flag;@489539f7,^[0-9]{3}\s{0,1}[0-9]{3}$]; default message [must match "^[0-9]{3}\s{0,1}[0-9]{3}$"]] [Field error in object 'contactDTO' on field 'lastName': rejected value [jk]; codes [Pattern.contactDTO.lastName,Pattern.lastName,Pattern.java.lang.String,Pattern]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [contactDTO.lastName,lastName]; arguments []; default message [lastName],[Ljavax.validation.constraints.Pattern$Flag;@489539f7,^[A-Z]{1}[a-zA-Z\s]{2,}$]; default message [Address book is LastNameInvalid]] [Field error in object 'contactDTO' on field 'email': rejected value [pihu123.gmail.com]; codes [Pattern.contactDTO.email,Pattern.email,Pattern.java.lang.String,Pattern]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [contactDTO.email,email]; arguments []; default message [email],[Ljavax.validation.constraints.Pattern$Flag;@489539f7,^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9]+(?:\.[a-zA-Z0-9]+)*$]; default message [must match "^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9]+(?:\.[a-zA-Z0-9]+)*$"]] [Field error in object 'contactDTO' on field 'firstName': rejected value [shanvika]; codes [Pattern.contactDTO.firstName,Pattern.firstName,Pattern.java.lang.String,Pattern]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [contactDTO.firstName,firstName]; arguments []; default message [firstName],[Ljavax.validation.constraints.Pattern$Flag;@489539f7,^[A-Z]{1}[a-zA-Z\s]{2,}$]; default message [Address book is FirstNameInvalid ]] [Field error in object 'contactDTO' on field 'phone': rejected value [7593450]; codes [Pattern.contactDTO.phone,Pattern.phone,Pattern.java.lang.String,Pattern]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [contactDTO.phone,phone]; arguments []; default message [phone],[Ljavax.validation.constraints.Pattern$Flag;@489539f7,^[0-9]{2}?[\s,-]{0,1}[7-9]{1}[0-9]{9}$]; default message [must match "^[0-9]{2}?[\s,-]{0,1}[7-9]{1}[0-9]{9}$"]] ]
2022-05-26 11:43:08.238  INFO 2736 --- [restartedMain] c.e.addressbook.AddressBookApplication   : Starting AddressBookApplication using Java 18.0.1.1 on Vishaka with PID 2736 (D:\SpringBoot\AddressBook\target\classes started by root in D:\SpringBoot\AddressBook)
2022-05-26 11:43:08.240 DEBUG 2736 --- [restartedMain] c.e.addressbook.AddressBookApplication   : Running with Spring Boot v2.7.0, Spring v5.3.20
2022-05-26 11:43:08.241  INFO 2736 --- [restartedMain] c.e.addressbook.AddressBookApplication   : The following 1 profile is active: "dev"
2022-05-26 11:43:09.028  INFO 2736 --- [restartedMain] c.e.addressbook.AddressBookApplication   : Started AddressBookApplication in 1.029 seconds (JVM running for 1.405)
2022-05-26 11:43:09.028  INFO 2736 --- [restartedMain] c.e.addressbook.AddressBookApplication   : AddressBook App Started in DEV Environment
2022-05-26 11:43:09.028  INFO 2736 --- [restartedMain] c.e.addressbook.AddressBookApplication   : AddressBook App DB User is root
2022-05-26 11:43:22.800  WARN 2736 --- [http-nio-8088-exec-1] .m.m.a.ExceptionHandlerExceptionResolver : Resolved [org.springframework.web.bind.MethodArgumentNotValidException: Validation failed for argument [0] in public org.springframework.http.ResponseEntity<com.example.addressbook.dto.ResponseDTO> com.example.addressbook.controller.AddressBookController.createContactData(com.example.addressbook.dto.ContactDTO) with 5 errors: [Field error in object 'contactDTO' on field 'firstName': rejected value [shanvika]; codes [Pattern.contactDTO.firstName,Pattern.firstName,Pattern.java.lang.String,Pattern]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [contactDTO.firstName,firstName]; arguments []; default message [firstName],[Ljavax.validation.constraints.Pattern$Flag;@36c7485b,^[A-Z]{1}[a-zA-Z\s]{2,}$]; default message [FirstNameInvalid ]] [Field error in object 'contactDTO' on field 'lastName': rejected value [jk]; codes [Pattern.contactDTO.lastName,Pattern.lastName,Pattern.java.lang.String,Pattern]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [contactDTO.lastName,lastName]; arguments []; default message [lastName],[Ljavax.validation.constraints.Pattern$Flag;@36c7485b,^[A-Z]{1}[a-zA-Z\s]{2,}$]; default message [ LastNameInvalid]] [Field error in object 'contactDTO' on field 'phone': rejected value [93450]; codes [Pattern.contactDTO.phone,Pattern.phone,Pattern.java.lang.String,Pattern]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [contactDTO.phone,phone]; arguments []; default message [phone],[Ljavax.validation.constraints.Pattern$Flag;@36c7485b,^[0-9]{2}?[\s,-]{0,1}[7-9]{1}[0-9]{9}$]; default message [Phone Number Invalid]] [Field error in object 'contactDTO' on field 'zip': rejected value [5000]; codes [Pattern.contactDTO.zip,Pattern.zip,Pattern.java.lang.String,Pattern]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [contactDTO.zip,zip]; arguments []; default message [zip],[Ljavax.validation.constraints.Pattern$Flag;@36c7485b,^[0-9]{3}\s{0,1}[0-9]{3}$]; default message [ZipCode Invalid]] [Field error in object 'contactDTO' on field 'email': rejected value [pihu123.gmail.com]; codes [Pattern.contactDTO.email,Pattern.email,Pattern.java.lang.String,Pattern]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [contactDTO.email,email]; arguments []; default message [email],[Ljavax.validation.constraints.Pattern$Flag;@36c7485b,^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9]+(?:\.[a-zA-Z0-9]+)*$]; default message [must match "^[a-zA-Z0-9_.+-]+@[a-zA-Z0-9]+(?:\.[a-zA-Z0-9]+)*$"]] ]
2022-05-26 11:44:07.051  INFO 552 --- [restartedMain] c.e.addressbook.AddressBookApplication   : Starting AddressBookApplication using Java 18.0.1.1 on Vishaka with PID 552 (D:\SpringBoot\AddressBook\target\classes started by root in D:\SpringBoot\AddressBook)
2022-05-26 11:44:07.053 DEBUG 552 --- [restartedMain] c.e.addressbook.AddressBookApplication   : Running with Spring Boot v2.7.0, Spring v5.3.20
2022-05-26 11:44:07.053  INFO 552 --- [restartedMain] c.e.addressbook.AddressBookApplication   : The following 1 profile is active: "dev"
2022-05-26 11:44:07.891  INFO 552 --- [restartedMain] c.e.addressbook.AddressBookApplication   : Started AddressBookApplication in 1.058 seconds (JVM running for 1.432)
2022-05-26 11:44:07.891  INFO 552 --- [restartedMain] c.e.addressbook.AddressBookApplication   : AddressBook App Started in DEV Environment
2022-05-26 11:44:07.891  INFO 552 --- [restartedMain] c.e.addressbook.AddressBookApplication   : AddressBook App DB User is root
2022-05-26 12:17:27.021  INFO 9356 --- [restartedMain] c.e.addressbook.AddressBookApplication   : Starting AddressBookApplication using Java 18.0.1.1 on Vishaka with PID 9356 (D:\SpringBoot\AddressBook\target\classes started by root in D:\SpringBoot\AddressBook)
2022-05-26 12:17:27.021 DEBUG 9356 --- [restartedMain] c.e.addressbook.AddressBookApplication   : Running with Spring Boot v2.7.0, Spring v5.3.20
2022-05-26 12:17:27.021  INFO 9356 --- [restartedMain] c.e.addressbook.AddressBookApplication   : The following 1 profile is active: "dev"
2022-05-26 12:17:27.844  INFO 9356 --- [restartedMain] c.e.addressbook.AddressBookApplication   : Started AddressBookApplication in 1.038 seconds (JVM running for 1.431)
2022-05-26 12:17:27.844  INFO 9356 --- [restartedMain] c.e.addressbook.AddressBookApplication   : AddressBook App Started in DEV Environment
2022-05-26 12:17:27.844  INFO 9356 --- [restartedMain] c.e.addressbook.AddressBookApplication   : AddressBook App DB User is root
2022-05-26 12:18:13.908  WARN 9356 --- [http-nio-8088-exec-4] .m.m.a.ExceptionHandlerExceptionResolver : Resolved [org.springframework.web.bind.MethodArgumentNotValidException: Validation failed for argument [0] in public org.springframework.http.ResponseEntity<com.example.addressbook.dto.ResponseDTO> com.example.addressbook.controller.AddressBookController.createContactData(com.example.addressbook.dto.ContactDTO) with 3 errors: [Field error in object 'contactDTO' on field 'address': rejected value [LB]; codes [Pattern.contactDTO.address,Pattern.address,Pattern.java.lang.String,Pattern]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [contactDTO.address,address]; arguments []; default message [address],[Ljavax.validation.constraints.Pattern$Flag;@b4b2626,^[A-Za-z,.0-9]{3,}$]; default message [Address is Invalid!!! ]] [Field error in object 'contactDTO' on field 'lastName': rejected value [Jk]; codes [Pattern.contactDTO.lastName,Pattern.lastName,Pattern.java.lang.String,Pattern]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [contactDTO.lastName,lastName]; arguments []; default message [lastName],[Ljavax.validation.constraints.Pattern$Flag;@b4b2626,^[A-Z]{1,}[a-zA-z\s]{2,}$]; default message [Address book is Invalid!!! ]] [Field error in object 'contactDTO' on field 'phone': rejected value [9345089378]; codes [Pattern.contactDTO.phone,Pattern.phone,Pattern.java.lang.String,Pattern]; arguments [org.springframework.context.support.DefaultMessageSourceResolvable: codes [contactDTO.phone,phone]; arguments []; default message [phone],[Ljavax.validation.constraints.Pattern$Flag;@b4b2626,^[0-9]{2}?[\s,-]{0,1}[7-9]{1}[0-9]{9}$]; default message [Invalid Phone number]] ]
2022-05-26 12:18:48.944  INFO 3192 --- [restartedMain] c.e.addressbook.AddressBookApplication   : Starting AddressBookApplication using Java 18.0.1.1 on Vishaka with PID 3192 (D:\SpringBoot\AddressBook\target\classes started by root in D:\SpringBoot\AddressBook)
2022-05-26 12:18:48.946 DEBUG 3192 --- [restartedMain] c.e.addressbook.AddressBookApplication   : Running with Spring Boot v2.7.0, Spring v5.3.20
2022-05-26 12:18:48.946  INFO 3192 --- [restartedMain] c.e.addressbook.AddressBookApplication   : The following 1 profile is active: "dev"
2022-05-26 12:18:49.737  INFO 3192 --- [restartedMain] c.e.addressbook.AddressBookApplication   : Started AddressBookApplication in 1.003 seconds (JVM running for 1.376)
2022-05-26 12:18:49.737  INFO 3192 --- [restartedMain] c.e.addressbook.AddressBookApplication   : AddressBook App Started in DEV Environment
2022-05-26 12:18:49.737  INFO 3192 --- [restartedMain] c.e.addressbook.AddressBookApplication   : AddressBook App DB User is root
